src/extension/entities/entitiesDataProvider.ts

```diff
@@ -75,6 +76,14 @@ export class EntityTreeDataProvider implements vscode.TreeDataProvider<EntityNod
   }
 }
 
+function entityIdToString(entityId: EntityId): string {
+  // Mask to get the lowest 32 bits.
+  const low = Number(BigInt(entityId) & 0xffffffffn);
```

Comment:
IMHO we can avoid unnecessary conversion to `number` here. This simplifies the code without performance penalties

```suggestion
  const low = BigInt(entityId) & 0xffffffffn;
```